cli/tests/pcluster/models/test_login_nodes_status.py

```diff
@@ -103,23 +132,44 @@ def test_full_login_nodes_status(self, mocker):
         mocker.patch("pcluster.aws.elb.ElbClient.__init__", return_value=None)
         mocker.patch(
             "pcluster.aws.elb.ElbClient.list_load_balancers",
-            return_value=[self.dummy_load_balancer_2, self.dummy_load_balancers],
+            return_value=[self.dummy_load_balancer_1, self.dummy_load_balancer_2, self.dummy_load_balancer_3],
         )
         mocker.patch("pcluster.aws.elb.ElbClient.describe_tags", return_value=self.dummy_tags_description)
         mocker.patch("pcluster.aws.elb.ElbClient.describe_target_groups", return_value=self.dummy_target_groups)
         mocker.patch("pcluster.aws.elb.ElbClient.describe_target_health", return_value=self.dummy_targets_health)
+
         login_nodes_status = LoginNodesStatus(self.dummy_stack_name)
-        login_nodes_status.retrieve_data(self.dummy_pool_name)
+        login_nodes_status.retrieve_data([self.dummy_pool_name_1, self.dummy_pool_name_2])
```

Comment:
you should include a check of `login_nodes_status.get_healthy_nodes(pool1)`

Reply:
checks added